models/requesters.js

```diff
@@ -59,15 +59,40 @@ const requesterModel = {
         : await db.runQuery(queries.getOne, [loanId, requesterId]);
       return {
         status: true,
-        data: loan.response.rows,
+        data: loan.response.rows
       };
     } catch (e) {
       return {
         status: false,
-        message: e,
+        message: e
       };
     }
   },
+  /**
+   *
+   * @author Grace Lungu
+   * @param {number} amount
+   */
+  async updateLoan(amount, id) {
+    try {
+      const loan = await db.runQuery(queries.updateLoan, [amount, id]);
+      if (loan.rowCount === 0) {
+        return {
+          status: false,
+          data: "The user does not have any pending loan"
```

Comment:
Would you also change `data` into `message` because what is returned in the controller when the `!response.status` is `message`?

Reply:
Okay, done;